serverless/networking/serverless-ossm-jwt.adoc

```diff
@@ -21,5 +21,10 @@ You can enable JSON Web Token (JWT) authentication for Knative services.
 Adding sidecar injection to pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
 ====
 
+[IMPORTANT]
+====
+You must set the annotation `sidecar.istio.io/rewriteAppHTTPProbers: "true"` in your Knative service as {ServerlessProductName} versions 1.14.0 and later use a HTTP probe as the readiness probe for Knative services by default.
```

Comment:
```suggestion
You must set the `sidecar.istio.io/rewriteAppHTTPProbers: "true"` annotation in your Knative service as {ServerlessProductName} versions 1.14.0 and later use an HTTP probe as the readiness probe for Knative services by default.
```

Reply:
Adding as a suggestion so you can just accept it @nak3 🙂

Reply:
Unless it doesn't work like that here, maybe I'm too used to upstream 😄

Reply:
Thank you! It makes very easy to fix :smile: